app/controllers/users/registrations_controller.rb

```diff
@@ -0,0 +1,60 @@
+class Users::RegistrationsController < Devise::RegistrationsController
```

Comment:
Style/ClassAndModuleChildren: Use nested module/class definitions instead of compact style.